src/components/ui/InfiniteList/index.tsx

```diff
@@ -0,0 +1,90 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import React, { useRef, useEffect, useState, HTMLAttributes, ReactNode } from 'react';
+
+import { StyledInfiniteList } from './Styled';
+import { BaseProps } from '../Base';
+import { Cog as Spinner } from '../icons'; // TODO: update icon
```

Comment:
I'm surprised that we don't already have a `spinner` icon. We should remedy that with design. 

One piece of feedback we have been getting from users is that for components such as this, which have both logic and some visual elements, they would like the option to pass in their own ui components. In this component we could include a prop for the spinner component, and the defaultProp could be this one. What do you think?